design/specs/column-extraction.md

```diff
@@ -8,87 +8,58 @@ editor: markdown
+### Selecting columns as a starting point
 
-2. **The user selects the columns (attributes) for the new entity**
+The user will select one or multiple columns to start with the column extraction process. All columns can be extracted, except for primary key columns. Once selected, the inspector panel will list 'Extract Columns to Table' as an action. Selecting this option will open the 'Extract Columns to Table' dialog.
 
-    The user chooses all the columns that will be attributes for the new object. For example, in the instance of the publisher, there could be columns for 'Publisher Name' and 'Publisher Id.' Including all the attributes is very important since columns cannot be moved after the fact.
+![image](/assets/design/specs/column-extraction/185139636-9a8a048c-9f87-4e26-aa2c-88f4e8d1648d.png)
 
-     <img width="1512" alt="image" src="https://user-images.githubusercontent.com/845767/177544077-9f074830-ce64-4ed9-80bc-53de53977f9c.png">
+## Extract Columns Dialog
 
+The dialog will list the columns that will be extracted and the table that will be created. The user can change the name of the new table. The dialog will also list the foreign key that will be created between the new table and the original table. The user can change the name of the foreign key.
 
-3. **The user clicks on the 'Extract Columns' option**
+![image](/assets/design/specs/column-extraction/185140900-ea1514c3-2423-42ad-ab98-a4f87848f463.png)
 
-    The user can choose to keep copies of the columns in the table if needed. By default the extracted columns are deleted.
+### Impact of the Extract Columns Operation
 
-4. **The user inspects the new foreign key column (link)**
+Since the split operation will affect the original table, the user should be made aware of the operation's impact. The dialog will make the following information clear:
 
-    The new foreign key column will contain links to the new records that have been created. Additionally, when selected, the inspector panel will display the link properties.
+- The original table will be updated to remove the extracted columns
+- A new table will be created with the extracted columns
+- A foreign key will be created between the new table and the original table
 
-    <img width="1469" alt="image" src="https://user-images.githubusercontent.com/845767/177546415-23b11ded-8987-4e69-ab1a-1decb9e773d3.png">
+### Link Column Name Convention
 
+To help users understand the role of the link column, the name of the link column will follow a convention that will be applied when users name the new table. The convention will be:
 
-5. **The user navigates to the new table**
+`table_name` `+` `Id`
```

Comment:
Can we please capitalize both letters? `ID`?

Also, what do we do if the name is already taken?

Reply:
In this [email thread](https://groups.google.com/a/mathesar.org/g/mathesar-developers/c/yu1dOjV7EC8) we decided to use the table name only. @ghislaineguerin can you update the specs to reflect that?

Reply:
I have updated the naming convention section.